### PR TITLE
Adopt text/template in node inspect

### PR DESCRIPTION
**Pull request by mkumatag**
**- What I did**
Changed the node inspect code to use text/template instead of using Sprintf. This was actually one of the TODO task in that file - https://github.com/docker/docker/blob/master/cli/command/node/inspect.go#L81

### CODE REVIEW

**Review by vdemeester — CHANGES_REQUESTED** [review]
I think the idea was more to mimic what we did for the other inspect command, like on `service inspect` https://github.com/docker/docker/blob/master/cli/command/service/inspect.go#L11 — reusing and completing the `cli/formatter` package 👼

**Comment by mkumatag** [comment]
@vdemeester make sense, spent some more time to understand `service inspect`. I could relate what I've submitted part of this PR. Let me refactor my code to align to `cli/formatter` package.

**Comment by mkumatag** [comment]
@vdemeester refactored the code using `cli/formatter`, PTAL at latest commit.

**Comment by cpuguy83** [comment]
```
11:16:16 --- FAIL: TestNodeInspectErrors (0.00s)
11:16:16 panic: runtime error: invalid memory address or nil pointer dereference [recovered]
11:16:16 	panic: runtime error: invalid memory address or nil pointer dereference
11:16:16 [signal SIGSEGV: segmentation violation code=0x1 addr=0xb0 pc=0x483a39]
11:16:16 
11:16:16 goroutine 13 [running]:
11:16:16 panic(0x7ccb40, 0xc4200100a0)
11:16:16 	/usr/local/go/src/runtime/panic.go:500 +0x1a1
11:16:16 testing.tRunner.func1(0xc42007ad80)
11:16:16 	/usr/local/go/src/testing/testing.go:579 +0x25d
11:16:16 panic(0x7ccb40, 0xc4200100a0)
11:16:16 	/usr/local/go/src/runtime/panic.go:458 +0x243
11:16:16 github.com/docker/docker/cli/command/node.runInspect(0xc353c0, 0xc42007af00, 0xc4202e7aa0, 0x1, 0x1, 0x0, 0x0, 0x0, 0xc4202e7aa0, 0x1)
```

**Comment by mkumatag** [comment]
@vdemeester all jobs got passed except windowsRS1 one, not sure whats wrong with it.!

**Comment by mkumatag** [comment]
@vdemeester ping

File: cli/command/formatter/node.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
+package formatter
+
+import (
+	"fmt"
+	"strings"
+
+	//	"github.com/docker/distribution/reference"
```
Comment (vdemeester) [inline cli/command/formatter/node.go]: Should remove the commented import/code 👼

**Review by vdemeester — COMMENTED** [review]
design LGTM
/cc @thaJeztah @dnephin @AkihiroSuda @cpuguy83

File: cli/command/formatter/node.go (hunk 1 above)
Comment (mkumatag) [inline cli/command/formatter/node.go]: @vdemeester fixed your comment and rebased the branch, PTAL into latest commit

**Review by vdemeester — APPROVED** [review]
LGTM (design & code) 🐻

**Review by dnephin — APPROVED** [review]
LGTM

**Pull request merged**